redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -31,23 +31,16 @@ object Input {
     def encode(data: Boolean): Chunk[String] = Chunk.single(wrap(if (data) "1" else "0"))
   }
 
-  case object BitFieldGetInput extends Input[BitFieldGet] {
-    def encode(data: BitFieldGet): Chunk[String] =
-      Chunk(wrap("GET"), wrap(data.`type`.stringify), wrap(data.offset.toString))
-  }
-
-  case object BitFieldSetInput extends Input[BitFieldSet] {
-    def encode(data: BitFieldSet): Chunk[String] =
-      Chunk(wrap("SET"), wrap(data.`type`.stringify), wrap(data.offset.toString), wrap(data.value.toString))
-  }
-
-  case object BitFieldIncrInput extends Input[BitFieldIncr] {
-    def encode(data: BitFieldIncr): Chunk[String] =
-      Chunk(wrap("INCRBY"), wrap(data.`type`.stringify), wrap(data.offset.toString), wrap(data.increment.toString))
-  }
-
-  case object BitFieldOverflowInput extends Input[BitFieldOverflow] {
-    def encode(data: BitFieldOverflow): Chunk[String] = Chunk(wrap("OVERFLOW"), wrap(data.stringify))
+  case object BitFieldCommandInput extends Input[BitFieldCommand] {
+    def encode(data: BitFieldCommand): Chunk[String] =
+      data match {
+        case BitFieldCommand.BitFieldGet(t, o)     => Chunk(wrap("GET"), wrap(t.stringify), wrap(o.toString))
+        case BitFieldCommand.BitFieldSet(t, o, v)  =>
+          Chunk(wrap("SET"), wrap(t.stringify), wrap(o.toString), wrap(v.toString))
+        case BitFieldCommand.BitFieldIncr(t, o, i) =>
+          Chunk(wrap("INCRBY"), wrap(t.stringify), wrap(o.toString), wrap(i.toString))
+        case bfo: BitFieldCommand.BitFieldOverflow => Chunk(wrap("OVERFLOW"), wrap(bfo.stringify))
+      }
```

Comment:
Let's add `import BitFieldCommand._` to this function, it'll make things slightly more readable.